common/utils/query_utils.ts

```diff
@@ -65,7 +66,13 @@ export const preprocessQuery = ({
 
   finalQuery = `${tokens![1]}=${
     tokens![2]
-  } | where ${timeField} >= '${start}' and ${timeField} <= '${end}'${tokens![3]}`;
+  } | where ${timeField} >= '${start}' and ${timeField} <= '${end}'`;
+
+  if (whereClause) {
+    finalQuery += ` AND ${whereClause}`;
+  }
```

Comment:
nice.

Reply:
this is temporary and will change when we support DSL in PPL where clauses